loafwallet.xcodeproj/project.pbxproj

```diff
@@ -3729,13 +3729,13 @@
 			buildPhases = (
 				825805FFD443BACDA18265D0 /* [CP] Check Pods Manifest.lock */,
 				2430679A238F538C00EBEA99 /* Update Localizable using BartyCrouch */,
+				583FE8702927ED5A009A6384 /* Run SwiftFormat */,
 				75A2A78C1DA5934300A983D8 /* Sources */,
 				75A2A78D1DA5934300A983D8 /* Frameworks */,
 				75A2A78E1DA5934300A983D8 /* Resources */,
 				75A2A8031DA5935F00A983D8 /* Embed App Extensions */,
 				22A9A9831DF63288000F0016 /* Embed Frameworks */,
 				C3A3FFB526FE46E8000FE955 /* Mark Dev Notes */,
-				58A9FE4829191E4700B75825 /* Check for unused code */,
```

Comment:
@Iferencak : So are you moving this functionality into the linter/formatter config?

Reply:
@kcw-grunt nice catch, It was accidentally removed, I added it back in.